src/arch/xtensa/boot_loader.c

```diff
@@ -206,8 +206,8 @@ static int32_t lp_sram_init(void)
 	/* add some delay before writing power registers */
 	idelay(delay_count);
 
-	lspgctl_value = shim_read(LSPGCTL);
-	shim_write(LSPGCTL, lspgctl_value & !LPSRAM_MASK(0));
+	lspgctl_value = shim_read(SHIM_LSPGCTL);
```

Comment:
It got a bit messy, we have LSPGCTL once used as shim (here) and another time as just memory mng register to turn it off at all (in asm_memory_management). Please clean it up.